### PR TITLE
[IMP] Add howto git, pr & github

### PR DESCRIPTION
**Pull request by 0ddlyoko**
Add a file that describes our Git infrastructure & how it works

### CODE REVIEW

**Review by Geolykt — CHANGES_REQUESTED** [review]

File: docs/howto_git.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    - These bugs will have "Non-Important" tag on GitHub
+    - When fixed, PR should target "develop" branch
+
+## Pull Request
+- Use minimal commit ! Squash & rebase your changes whenever you edit your PR before submitting ! Don't do like [this](https://github.com/tazz4843/McPy/pull/20) guy !
+- We use some pefixed tags in our PR:
+  - [FIX]: When your PR fix a bug
+  - [ADD]: When your PR add a new feature
+  - [IMP]: When your PR improve something that exists
+- More informations about Squash & rebase [here](https://www.devroom.io/2011/07/05/git-squash-your-latests-commits-into-one/) or [here (french)](https://www.ekino.com/articles/comment-squasher-efficacement-ses-commits-avec-git)
```
Comment (Geolykt) [inline docs/howto_git.md]: It would be better if an english articely is linked here.
Otherwise, no malicious code (yet)

**Review by Geolykt — APPROVED** [review]

File: docs/howto_git.md (hunk 1 above)
Comment (0ddlyoko) [reply docs/howto_git.md]: Do you have an english link? All link I have is in french

Comment (0ddlyoko) [reply docs/howto_git.md]: Found, I update the link

**Comment by Geolykt** [comment]
Pushed due to no recommendations in the past 7 days

**Pull request merged**